base/vcpkg.json

```diff
@@ -1,58 +1,63 @@
 {
-  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg/master/scripts/vcpkg.schema.json",
-  "name": "apra-pipes-cuda",
-  "version": "0.0.1",
-  "builtin-baseline": "356814e3b10f457f01d9dfdc45e1b2cac0ff6b60",
-  "dependencies": [
-    {
-      "name": "opencv4",
-      "default-features": false,
-      "features": [
+    "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg/master/scripts/vcpkg.schema.json",
+    "name": "apra-pipes-cuda",
+    "version": "0.0.1",
+    "builtin-baseline": "86811c9b87c81b533c4de069a6a30c59a7daf6cc",
+    "overrides": [
+      {
+        "name": "whisper",
+        "version": "1.5.2"
+      }
+    ],
+    "dependencies": [
+      {
+        "name": "opencv4",
+        "default-features": false,
+        "features": [
           "contrib",
-          "cuda",
-          "cudnn",
```

Comment:
These two lines should not be removed, I think this is a side effect of using fix-vcpkg.bat

Reply:
done